.github/workflows/minimal.yml

```diff
@@ -0,0 +1,51 @@
+on:
+  push:
+    branches: [master]
+  pull_request:
+name: With dependencies at minimal versions
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+      - uses: actions/checkout@v2
+      - run: |
+          chmod 600 .test-key
+          ssh -i .test-key -v -p 2222 -l test-user 127.0.0.1 -o StrictHostKeyChecking=accept-new whoami
+        name: Test ssh connectivity
+      - run: |
+          eval $(ssh-agent)
+          echo "SSH_AUTH_SOCK=$SSH_AUTH_SOCK" >> $GITHUB_ENV
+          echo "SSH_AGENT_PID=$SSH_AGENT_PID" >> $GITHUB_ENV
+          cat .test-key | ssh-add -
+        name: Set up ssh-agent
+      - name: cargo update -Zminimal-versions
+        uses: actions-rs/cargo@v1
+        with:
+          command: update
+          toolchain: nightly
+          args: -Zminimal-versions
+      - name: cargo test
+        uses: actions-rs/cargo@v1
+        with:
+          command: test
+          args: --all-features
+        env:
+          # makes all the ignored tests not ignored
+          RUSTFLAGS: --cfg=ci
+    services:
+      openssh:
+        image: linuxserver/openssh-server:amd64-latest
+        ports:
+          - 2222:2222
+        env:
```

Comment:
`linuxserver/openssh-server` does not enable ssh port forwarding by default, it requires an additional environment variable to be set:

```
DOCKER_MODS='linuxserver/mods:openssh-server-ssh-tunnel'
```

Reply:
> but am running into test failures for local_socket_forward and remote_socket_forward. The [failures are](https://github.com/jonhoo/openssh-rs/runs/5266473147?check_suite_focus=true#step:7:152):

Setting `DOCKER_MODS` here would fix the test failures @jonhoo